packages/vats/scripts/restart-vats.js

```diff
@@ -0,0 +1,23 @@
+import { makeHelpers } from '@agoric/deploy-script-support';
+
+/** @type {import('@agoric/deploy-script-support/src/externalTypes.js').ProposalBuilder} */
```

Comment:
:clap: :tada: thanks so much for adding static types around here! Sorry we didn't do it sooner.